packages/psss-server/src/connections/AuthConnection.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { AuthResponse, Credentials } from '../types';
+import { ApiConnection } from './ApiConnection';
+
+const {
+  MEDITRAK_API_CLIENT_NAME,
+  MEDITRAK_API_CLIENT_PASSWORD,
+  MEDITRAK_API_URL = 'http://localhost:8090/v2',
+} = process.env;
+
+const basicAuthHandler = {
+  getAuthHeader: async () => {
+    const credentials = `${MEDITRAK_API_CLIENT_NAME}:${MEDITRAK_API_CLIENT_PASSWORD}`;
+    return `Basic ${Buffer.from(credentials).toString('base64')}`;
```

Comment:
Detail, but this can also be a constant in the outer scope

Reply:
True. I'm not sure I have a preference either way, so I've swapped it to your suggestion